.github/workflows/gradle.yml

```diff
@@ -0,0 +1,53 @@
+name: Java CI with Maven
+
+on:
+  push:
+    branches: [ rails_2_develop ]
+  pull_request:
+    branches: [ rails_2_develop ]
+
+jobs:
+  jdk11:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-latest, windows-latest, ubuntu-latest]
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-java@v1
+      with:
+        java-version: 11
+        java-package: jdk
+    - name: Tests - OpenJDK 11 
+      run: gradle clean test
+
+  ubuntu:
+    name: Build - Ubuntu
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Setup Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: 14
+          java-package: jdk
+      - name: Execute Tests
+        uses: eskatos/gradle-command-action@v1
+        with:
+          gradle-version: 6.3
+          arguments: clean test
+      - name: Build jar
+        uses: eskatos/gradle-command-action@v1
+        with:
+          gradle-version: 6.3
+          arguments: clean build
```

Comment:
We have two choices how to use `gradle` with GitHub actions:
1.  we use `github-command-action`
2.  we use our wrapper `./gradlew`

The first option is follows the aspired style for GitHub Actions. The second option allows us to specify the Gradle version in a single place instead of in two.

@neutronc @jklap which option do you think is better?

Reply:
I'd lean towards the second option-- if we expect developers to use the wrapper then having GitHub actions use it also makes sure it all works